corehq/apps/cleanup/management/commands/evaluate_couch_model_for_sql.py

```diff
@@ -295,13 +299,15 @@ def generate_management_command(self):
             models_path=self.models_path,
             db_slug=db_slug,
             dates_import=dates_import,
-            suggested_updates=suggested_updates
+            suggested_updates=suggested_updates,
+            submodels=submodels
         )
 
     def is_submodel_key(self, key):
-        if self.field_types[key] in (self.FIELD_TYPE_SUBMODEL_LIST, self.FIELD_TYPE_SUBMODEL_DICT):
-            return True
-        return "." in key
+        return "." in key or self.is_field_type_submodule(key)
+
+    def is_field_type_submodule(self, key):
```

Comment:
I'm not totally following the difference between `submodel` and `submodule`, are they different?

Reply:
sorry, that's a typo